compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.cpp

```diff
@@ -464,6 +465,52 @@ FailureOr<Flow::DispatchRegionOp> Flow::wrapOpInDispatchRegion(
   return newRegionOp;
 }
 
+//===---------------------------------------------------------------------===//
+// UnCSE Clonable ops
+//===---------------------------------------------------------------------===//
+
+namespace {
+struct UnCseClonableOps : public RewritePattern {
+  UnCseClonableOps(MLIRContext *context)
+      : RewritePattern(MatchAnyOpTypeTag(), /*benefit=*/1, context) {}
+
+  LogicalResult matchAndRewrite(Operation *op,
+                                PatternRewriter &rewriter) const override {
+    if (!isa<FunctionOpInterface>(op->getParentOp())) {
+      return rewriter.notifyMatchFailure(op, "parent is not a func.func");
+    }
+    if (!Flow::isClonableIntoDispatchOp(op)) {
+      return rewriter.notifyMatchFailure(op, "not clonable op");
+    }
+
+    if (llvm::hasSingleElement(op->getUsers())) {
```

Comment:
`op->hasOneUse()`

Reply:
I want a single `user`. There is no `op->hasOneUser()`... the same value could have multiple uses in a consumer which will pessimize things.

Reply:
Removed all of this